src/java.base/share/classes/com/sun/crypto/provider/CipherCore.java

```diff
@@ -564,48 +564,48 @@ void init(int opmode, Key key, AlgorithmParameterSpec params,
         if (params != null) {
             if (cipherMode == GCM_MODE) {
                 if (params instanceof GCMParameterSpec) {
-                    tagLen = ((GCMParameterSpec)params).getTLen();
+                    tagLen = ((GCMParameterSpec) params).getTLen();
```

Comment:
This is missing the `try` block added upstream by
* 8258915: Temporary buffer cleanup

Reply:
Fixed. I probably compared a wrong branch and thought this `try` related change was removed by previous JCL patch.
Please have another look.